apps/meteor/client/startup/notifications/calendar.ts

```diff
@@ -0,0 +1,40 @@
+import type { IUser, ICalendarNotification } from '@rocket.chat/core-typings';
```

Comment:
could we refactor this entire file to react/hooks? also only loading the file just after the user gets logged would be great

Reply:
moved to konchat